src/internal_model/lomse_im_note.cpp

```diff
@@ -220,6 +220,7 @@ ImoNote::ImoNote(int step, int octave, int noteType, EAccidentals accidentals, i
     , m_stemDirection(stem)
     , m_pTieNext(nullptr)
     , m_pTiePrev(nullptr)
+    , m_computedStem(k_computed_stem_undecided)
```

Comment:
This is not exactly necessary to fix the crash as this constructor doesn't seem to be used when reading a MusicXML file. However it is better not to skip initialization of this variable.

Reply:
Agree. Better initialize always all variables